sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/LockRenewalOperation.java

```diff
@@ -49,6 +53,34 @@
         this(lockToken, maxLockRenewalDuration, isSession, renewalOperation, OffsetDateTime.now());
     }
 
+    /**
```

Comment:
Do we need all these overloads? I'm not entirely sure why we need this consumer... Can't we map the `renewalOperation.map(update -> {});`?